packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -0,0 +1,237 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+
+$mdc-grid-list-tile-width: 200px;
+$mdc-grid-list-tile-secondary-padding: 16px;
+$mdc-grid-list-tile-secondary-padding-narrow: 8px;
+$mdc-grid-list-oneline-cap-secondary-height: 48px;
+$mdc-grid-list-twoline-cap-secondary-height: 68px;
+$mdc-grid-list-tile-secondary-icon-size: 24px;
+
+// postcss-bem-linter: define grid-list
+// stylelint-disable plugin/selector-bem-pattern
+@mixin mdc-grid-list--tile-aspect($width-height-ratio) {
+  .mdc-grid-tile .mdc-grid-tile__primary {
+    padding-bottom: calc(100% / #{$width-height-ratio});
+  }
+}
+
+@mixin mdc-grid-list--tile-gutter($gutter-width) {
+  .mdc-grid-tile {
+    margin: $gutter-width/2 0;
+    padding: 0 $gutter-width/2;
+
+    &__secondary {
+      left: $gutter-width/2;
+      width: calc(100% - #{$gutter-width});
+    }
+  }
+
+  .mdc-grid-list__tiles {
+    margin: $gutter-width/2 auto;
+  }
+}
+// stylelint-enable plugin/selector-bem-pattern
+
+.mdc-grid-list {
+  /*
+   *  Default grid list style:
+   *  - mdc-grid-list--tile-gutter-4
+   *  - mdc-grid-list--tile-aspect-1x1
+   *  - mdc-grid-list--oneline-caption
+   *  - mdc-grid-list--footer-caption
+   *  - mdc-grid-list--no-icon
+   *
+   */
+  @include mdc-grid-list--tile-aspect(1);
+  @include mdc-grid-list--tile-gutter(4px);
+
+  // stylelint-disable plugin/selector-bem-pattern
+  .mdc-grid-tile {
+    &__secondary {
+      bottom: 0;
+      height: $mdc-grid-list-oneline-cap-secondary-height;
+      padding: $mdc-grid-list-tile-secondary-padding;
+    }
+
+    &__icon {
+      font-size: 0;
+    }
+  }
+  // stylelint-enable plugin/selector-bem-pattern
+
+  &__tiles {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+// stylelint-disable plugin/selector-bem-pattern
```

Comment:
I'd maybe include a comment here as to why the bem pattern plugin needs to be disabled, as it's not immediately apparent from looking at the code.